Makefile

```diff
@@ -3,6 +3,7 @@ appname := mqtt-stresser
 sources := $(wildcard *.go)
 
 build = GOOS=$(1) GOARCH=$(2) go build -o build/$(appname)$(3)
+static-build = CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o build/$(appname) .
```

Comment:
static GOOS var?

Reply:
what do you think about

```
static-build = CGO_ENABLED=0 GOOS=$(1) GOARCH=$(2) go build -a -installsuffix cgo -o build/$(appname)$(3) .
```

?

Reply:
Why introduce a new indirection for only one build target? IMHO that is over the top.

Reply:
why hardcoding linux when there are other platforms too? it would be great if you add darwin-static and windows-static, too! At least to keep it consistent :)